eng/pipelines/coreclr/templates/helix-queues-setup.yml

```diff
@@ -107,7 +107,7 @@ jobs:
     # OSX arm64
     - ${{ if eq(parameters.platform, 'OSX_arm64') }}:
       - ${{ if and(eq(variables['System.TeamProject'], 'public'), in(parameters.jobParameters.helixQueueGroup, 'pr', 'ci', 'libraries')) }}:
-        - OSX.1100.ARM64.Open
+        - OSX.1500.ARM64.Open
```

Comment:
What is 1500? I assumed that on Arm64, macOS followed the same version numbering -- ie macOS 11 or macOS12. Is that not right?

Reply:
Yeah, it is mistake, it should be 1200